apps/api/src/app/events/services/chat-service/chat.factory.ts

```diff
@@ -7,7 +7,7 @@ import { MSTeamsHandler } from './handlers/msteams.handler';
 export class ChatFactory implements IChatFactory {
   handlers: IChatHandler[] = [new SlackHandler(), new DiscordHandler(), new MSTeamsHandler()];
 
-  getHandler(integration: IntegrationEntity): IChatHandler {
+  getHandler(integration: IntegrationEntity) {
```

Comment:
What's the reason behind removing some of the return types?

Reply:
the functions are defined in the interface `IChatFactory`, so you don't have to redefine them here